3-networks-hub-and-spoke/modules/restricted_shared_vpc/main.tf

```diff
@@ -130,7 +130,10 @@ module "region1_router1" {
   bgp = {
     asn                  = var.bgp_asn_subnet
     advertised_groups    = ["ALL_SUBNETS"]
-    advertised_ip_ranges = [{ range = local.restricted_googleapis_cidr }]
+    advertised_ip_ranges = [
+      { range = local.restricted_googleapis_cidr },
+      { range = "35.199.192.0/19" }
```

Comment:
`{ range = "35.199.192.0/19" }` is conditional on the network being the one that  will access the dns server in the on-prem infrastructure